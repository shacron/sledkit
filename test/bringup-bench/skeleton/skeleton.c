#include "libmin.h"

int
main(void)
{
  libmin_printf("This is a test!, %d, %f...\n", 23, 44.4);

  libmin_success();
  return 0;
}


