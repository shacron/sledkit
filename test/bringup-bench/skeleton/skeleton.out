This is a test!, 23, 44.400000...
